src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -1135,31 +1135,40 @@ BOOL MethodTableBuilder::CheckIfSIMDAndUpdateSize()
 
     LPCUTF8 className;
     LPCUTF8 nameSpace;
+
     if (FAILED(GetMDImport()->GetNameOfTypeDef(bmtInternal->pType->GetTypeDefToken(), &className, &nameSpace)))
         return false;
 
     if (strcmp(className, "Vector`1") != 0 || strcmp(nameSpace, "System.Numerics") != 0)
         return false;
 
-    if (!TargetHasAVXSupport())
-        return false;
-
     EEJitManager *jitMgr = ExecutionManager::GetEEJitManager();
+
     if (jitMgr->LoadJIT())
```

Comment:
It should not be necessary to load the JIT for this. (Initialization of the flags in EEJitManager can be done earlier.)

Reply:
As in the initialization will have already been done or I need to ensure its already been done?
-- Just trying to determine if I can just remove this `if` or if I need to do some other work on top as well.

Reply:
- Delete `LoadJIT` call here
- Move `SetCpuInfo()` in codeman.cpp from the LoadJIT method to EEJitManager constructor

Reply:
Fixed.